### PR TITLE
Allow to remove branch on integration

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: scripts/delete_branch.sh [hunk 1]
```diff
@@ -1,44 +1,64 @@
...
-if [ "$#" -ne 1 ]; then
-        echo "No branch name given"
-        echo "Possible values:"
-        for b in ${BRANCHES};
-            do
-               echo ${b}
-            done
-        exit 0
+if [ $# -ne 2 ]; then
+    echo "No branch name given, use parameter BRANCH_TO_DELETE=your_branch_name"
```
Comment (procrastinatio) [inline scripts/delete_branch.sh]: Message is misleading: yo want two parameters

Comment (procrastinatio) [reply scripts/delete_branch.sh]: And I would also test the number of args at the very top

Comment (oterral) [reply scripts/delete_branch.sh]: I can't do it at the very top I need all the previous variables to display the list of branches available

**Comment by oterral** [comment]
@procrastinatio doc improved

File: scripts/delete_branch.sh [hunk 2]
```diff
@@ -1,44 +1,71 @@
 #!/bin/bash
```
Comment (loicgasser) [inline scripts/delete_branch.sh]: we could add set -e

**Pull request merged**